src/zenml/zen_stores/migrations/versions/ade72effebaf_added_logs_table.py

```diff
@@ -0,0 +1,62 @@
+"""Added logs table [ade72effebaf].
+
+Revision ID: ade72effebaf
+Revises: 0.39.1
+Create Date: 2023-05-10 17:26:03.716357
+
+"""
+import sqlalchemy as sa
+import sqlmodel
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "ade72effebaf"
+down_revision = "0.39.1"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    """Upgrade database schema and/or data, creating a new revision."""
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "logs",
+        sa.Column(
+            "pipeline_run_id", sqlmodel.sql.sqltypes.GUID(), nullable=True
+        ),
+        sa.Column("step_run_id", sqlmodel.sql.sqltypes.GUID(), nullable=True),
+        sa.Column(
+            "artifact_store_id", sqlmodel.sql.sqltypes.GUID(), nullable=True
```

Comment:
How come this is still `nullabe=True`? You changed it in the schema, I wonder why alembic didn't pick up on it
```suggestion
            "artifact_store_id", sqlmodel.sql.sqltypes.GUID(), nullable=False
```